Makefile

```diff
@@ -303,6 +303,9 @@ install: runtime
 	$(ENV_INSTALL) -d $(ENV_INST_LUADIR)/apisix/plugins/limit-count
 	$(ENV_INSTALL) apisix/plugins/limit-count/*.lua $(ENV_INST_LUADIR)/apisix/plugins/limit-count/
 
+	$(ENV_INSTALL) -d $(ENV_INST_LUADIR)/apisix/plugins/google-logging
+	$(ENV_INSTALL) apisix/plugins/google-logging/*.lua $(ENV_INST_LUADIR)/apisix/plugins/google-logging/
```

Comment:
We should use the name `google-cloud-logging` instead of google-logging? It is confusing if we don't mention which google product we are working with.

Reply:
Are you referring to the name of the directory or the name of the plugin?

Reply:
Both.

Reply:
updated